EIPS/eip-5496.md

```diff
@@ -76,7 +76,7 @@ interface IERC721Cloneable {
     /// @param tokenId The identifier of the queried NFT
     /// @param privilegeId The identifier of the queried privilege
     /// @param referrer The address of the referrer
```

Comment:
could you elaborate of what a referrer is in this context?

Reply:
For example, a local pizza shop offers a 30% off Coupon and the owner of the shop encourages their consumers to share the coupon with friends, then the friends can get the coupon. Let's say Tom gets 30% off Coupon from the shop and he shares the coupon with Alice. Alice gets the coupon too and Alice's referrer is Tom.  For some certain cases, Tom may get more rewards from the shop. This's a good way to expand markets for merchants.

Reply:
This is good content! It should go into the EIP itself.

Reply:
done